Extractor/Config/SubscriptionConfig.cs

```diff
@@ -105,6 +106,17 @@ public class SubscriptionConfig : SubscriptionInstanceConfig
         /// Recreate subscriptions that have stopped publishing. True by default.
         /// </summary>
         public bool RecreateStoppedSubscriptions { get; set; } = true;
+
+        /// <summary>
+        /// Optional grace period for recreating stopped subscriptions.
+        /// Defaults to 8 * Publishing interval
+        /// </summary>
+        public string RecreateSubscriptionGrace
```

Comment:
Can we call it ResubscriptionGracePeriod or something else that includes Period in the name?